manifests/install.pp

```diff
@@ -75,13 +75,20 @@
     $_package_options = {}
   }
 
+  if $::osfamily =~ /windows/ {
```

Comment:
Yeah I was thinking about that. It seemed redundant to pass it in alongside the package_file_name, but the reduction in duplicated logic may make it worth it.